src/control/server/ioserver/exec.go

```diff
@@ -139,16 +146,23 @@ func (r *Runner) Start(ctx context.Context, errOut chan<- error) error {
 	return nil
 }
 
-func (r *Runner) setStarted() {
-	atomic.StoreUint32(&r.started, 1)
+func (r *Runner) setRunning() {
+	atomic.StoreUint32(&r.running, 1)
 }
 
 func (r *Runner) setStopped() {
-	atomic.StoreUint32(&r.started, 0)
+	atomic.StoreUint32(&r.running, 0)
 }
 
-func (r *Runner) IsStarted() bool {
-	return atomic.LoadUint32(&r.started) == 1
+func (r *Runner) IsRunning() bool {
+	return atomic.LoadUint32(&r.running) == 1
```

Comment:
Kris made a good point in a different PR that a more resilient pattern for these would be to check that the value != 0. In other words, it returns false if 0, true if anything else.